birdhouse/optional-components/README.md

```diff
@@ -112,3 +112,26 @@ Magpie will be automatically configured to give complete public anonymous
 access for this WPS service.
 
 Canarie monitoring will also be automatically configured for this WPS service.
+
+
+## All access given to test user
```

Comment:
Suggested text change: "Give public access to all resources for testing purposes"